config.yml.example

```diff
@@ -0,0 +1,69 @@
+DiscordBotToken: ""
+GuildID: ""
+# The ID of the channel that the bot will send its welcome message.
+PrimaryChannelID: ""
+AlertingMethods:
+  - Twilio:
```

Comment:
replace "- " with ""

Reply:
also rename file to example.config.yml

Reply:
Still need to rename the file so we get syntax highlighting in VSCode. @Xanzibaer